[editor-common] anchor - add config to handle element preview (#2293)
wix/ricos: website/docs/plugins_api/LinkPlugin.mdx

> ### `linkTypes`

Reviewer: ```suggestion
### `Anchor`
```

}
```

### `previewElement`

> anchor configuration for customize the preview element that shown in anchor panel.

Reviewer: ```suggestion
Anchor configuration to customize the preview elements shown in the anchor panel.
```

}
```

### `blockPreview`

Anchor configuration to customize the preview elements shown in the anchor panel.

example:

```jsx
{
  linkTypes: {
      anchor: {
        blockPreview: ({ anchorType, data, text, type }) => {
          console.log({ anchorType, data, text, type });
          const blockPreview = { thumbnail: <div>bla1</div>, type: 'bla2', content: 'bla3' };
          return blockPreview;
        },
      },
    },
}
```

`anchorType` - block plugin type e.g. "wix-draft-plugin-image", "wix-draft-plugin-video", unstyled...
`data` - block entity data e.g. src, config...
`text` - block text

> `type` - block type e.g. "atomic", "unstyled

Reviewer: is there need for both anchorType and type? 
Probably anchorType is enough? But in the api it's better called type?
